docs/navigation-parameters-reference.md

```diff
@@ -129,6 +129,30 @@ Node parameters are all the parameters that can be added to an individual naviga
 - **type**: string
 - **description**: sets the child node that Luigi activates automatically if the current node has no **viewUrl** defined. Provide **pathSegment** of the child node you want to activate as a string.
 
+<!-- add-attribute:class:success -->
+>**NOTE:** To define **defaultChildNode** on root level to specify a different default node than the first header navigation node, set an object with **defaultChildNode** and **children** definition as `navigation.nodes` instead of an array of nodes.
```

Comment:
```suggestion
>**NOTE:** To define a root-level **defaultChildNode** which is different than the first header navigation node, set an object with **defaultChildNode** and **children** set to `navigation.nodes` instead of an array of nodes.
```